### PR TITLE
fix: clear previous validation error

### PR DESCRIPTION
**Pull request by foreleven**

### CODE REVIEW

File: example/src/UserForm.tsx [hunk 1]
```diff
@@ -9,17 +9,13 @@ export const UserForm = () => {
...
-      e.preventDefault();
-      console.log(model);
-      const valid = await model.validate('user');
-      if (valid) {
-        alert('valid success');
-      }
-    },
-    [model],
-  );
+  const handleSubmit = async e => {
```
Comment (JounQin) [inline example/src/UserForm.tsx]: 这个 `useCallback` 不应该去掉吧？

Comment (foreleven) [reply example/src/UserForm.tsx]: Dan said that it is not necessary 😄

Comment (JounQin) [reply example/src/UserForm.tsx]: Any reference URL, bro?

File: tsconfig.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "paths": {
       "stated-form-bean": ["src"]
     },
-    "sourceMap": true,
+    "inlineSourceMap": true,
```
Comment (JounQin) [inline tsconfig.json]: Why change this?

File: example/src/UserForm.tsx [hunk 3]
```diff
@@ -9,17 +9,13 @@ export const UserForm = () => {
...
+    e.preventDefault();
+    const valid = await model.validate('user');
+    if (valid) {
+      console.info('valid success');
+    }
+  };
   return (
     <div>
```
Comment (JounQin) [inline example/src/UserForm.tsx]: The div wrapper is redundant.

**Pull request merged**